blockchain/utxobackend.go

```diff
@@ -229,6 +296,213 @@ func (l *LevelDbUtxoBackend) FetchStats() (*UtxoStats, error) {
 	return stats, nil
 }
 
+// dbPutUtxoDatabaseInfo uses an existing database transaction to store the
+// database information.
+func (f *LevelDbUtxoBackend) dbPutUtxoDatabaseInfo(dbTx database.Tx,
+	info *utxoBackendInfo) error {
+
+	// uint32Bytes is a helper function to convert a uint32 to a byte slice
+	// using the byte order specified by the database namespace.
+	uint32Bytes := func(ui32 uint32) []byte {
+		var b [4]byte
+		byteOrder.PutUint32(b[:], ui32)
+		return b[:]
+	}
+
+	// uint64Bytes is a helper function to convert a uint64 to a byte slice
+	// using the byte order specified by the database namespace.
+	uint64Bytes := func(ui64 uint64) []byte {
+		var b [8]byte
+		byteOrder.PutUint64(b[:], ui64)
+		return b[:]
+	}
+
+	// Store the database version.
+	meta := dbTx.Metadata()
+	bucket := meta.Bucket(utxoDbInfoBucketName)
+	err := bucket.Put(utxoDbInfoVersionKeyName, uint32Bytes(info.version))
+	if err != nil {
+		return err
+	}
+
+	// Store the compression version.
+	err = bucket.Put(utxoDbInfoCompressionVerKeyName, uint32Bytes(info.compVer))
+	if err != nil {
+		return err
+	}
+
+	// Store the UTXO set version.
+	err = bucket.Put(utxoDbInfoUtxoVerKeyName, uint32Bytes(info.utxoVer))
+	if err != nil {
+		return err
+	}
+
+	// Store the database creation date.
+	return bucket.Put(utxoDbInfoCreatedKeyName,
+		uint64Bytes(uint64(info.created.Unix())))
+}
+
+// dbFetchUtxoDatabaseInfo uses an existing database transaction to fetch the
+// database versioning and creation information.
+func (f *LevelDbUtxoBackend) dbFetchUtxoDatabaseInfo(dbTx database.Tx) *utxoBackendInfo {
```

Comment:
Similarly, how about `dbFetchUtxoBackendInfo` to match?

Reply:
Updated.